.changeset/short-kids-hunt.md

```diff
@@ -0,0 +1,20 @@
+---
+"wrangler": minor
```

Comment:
We will mark every changes as patch as Pipeline is still in closed beta.
```suggestion
"wrangler": patch
```

Reply:
Sounds good. Thanks @edmundhung !